### PR TITLE
fix : speaker's image ui fix (fix :2067)

### LINKED ISSUE #2067: The Speaker Image in Collapsing Toolbar layout does not cover full frame layout
**Actual Behaviour**

The Speaker Image in Collapsing Toolbar layout does not cover full frame layout.

**Expected Behaviour**

The Speaker Image in Collapsing Toolbar layout should cover full frame layout.

**Would you like to work on the issue?**
Yes
PR -> #2065

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #2067

Changes: The speaker image in collapsing toolbar region in activity_speakers now covers the full frame layout.

Screenshots for the change: 
screenshot_20171226-210151

### CODE REVIEW

**Comment by dr0pdb** [comment]
@dreadpool2 Please create an issue for this PR and then link this PR to the created issue :smile:

**Comment by mayank8318** [comment]
@dreadpool2 Please add the issue number in the PR comment as ->  Fix #....

**Comment by dr0pdb** [comment]
@dreadpool2 Please add Fixes #... in your PR comment, not in the PR title.
It automatically closes the issue once the PR is merged. You can read about it [here](https://github.com/blog/1506-closing-issues-via-pull-requests)

File: android/app/src/main/res/layout/activity_speakers.xml [hunk 1]
```diff
@@ -30,13 +30,13 @@
 
             <FrameLayout
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
+                android:layout_height="@dimen/item_image_view_extra_large"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/activity_speakers.xml]: Using static height for CollapsingToolbar is alright, but doing so everywhere is not. Match Parent should be sufficient for the behaviour required here

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by dreadpool2** [comment]
@iamareebjamal Done the change!

**Comment by dreadpool2** [comment]
@iamareebjamal  did the changes

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**